corehq/apps/users/templates/users/edit_commcare_user.html

```diff
@@ -70,11 +70,15 @@
           <div class="form form-horizontal">
             <fieldset>
               <legend>{% trans 'Force Full Refresh on next Mobile Sync' %}</legend>
-              <div>
+              <form action="{% url "force_user_412" domain couch_user.user_id %}" method="post">
                 <p>
-                  <a class="btn btn-primary {% if not has_any_sync_logs %} disabled{% endif %}" href="#force_user_412_{{ couch_user.user_id }}" data-toggle="modal" {% if not has_any_sync_logs %}aria-disabled="true"{% endif %}>
+                  See <a href="https://confluence.dimagi.com/display/commcarepublic/CommCare+Android+Troubleshooting#CommCareAndroidTroubleshooting-ForceFullRefresh">{% trans "Force Full Refresh" %}</a> for more information.
+                </p>
+                {% csrf_token %}
+                <p>
+                  <button type="submit" class="btn btn-primary disable-on-submit-no-spinner{% if not has_any_sync_logs %} disabled{% endif %}" {% if not has_any_sync_logs %}disabled{% endif %}>
```

Comment:
you don't like spinners?

Reply:
Well, this is using the same icon as the button's icon (meaning "refresh"), so it was weird to then add another one of them next to it that was spinning.

Reply:
ah yeah...we do that in some places but I don't love it either, and it always seems like a disproportionate amount of work to replace the existing icon